jsapp/scss/libs/_mdl.scss

```diff
@@ -1,10 +1,13 @@
 // Theming variables, mixins and includes
 
 $black: #414453;
+$cool-charcoal: lighten($black, 10%);
```

Comment:
I'm not sure that these variables are better than using the SCSS mixins directly... maybe it's just a personal preference.